core/src/main/java/org/mskcc/cbio/portal/scripts/ImportSVData.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.mskcc.cbio.portal.scripts;
+
+import org.mskcc.cbio.portal.dao.*;
+import org.mskcc.cbio.portal.util.*;
+import org.cbioportal.service.SVService;
+import org.cbioportal.model.SV;
+import org.cbioportal.persistence.mybatis.SVMapper;
+
+import java.io.*;
+import java.util.*;
+
+import org.springframework.stereotype.Service;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.transaction.annotation.Transactional;
+
+/**
+ *
+ * @author jake
+ */
+public class ImportSVData {
+    
+    private File svFile;
+    private String geneticProfileStableId;
+    
+    @Autowired
+    private SVMapper svMapper;
+    
+    @Autowired
+    private SVService svService;
+    
+    @Transactional
+    public void insertSV(String sampleId, String annotation, String breakpoint_type, String comments,
```

Comment:
I'm not sure that the import method was optimized, I was more just bootlegging the old import to get data in since I thought you were refactoring the imports